Documentation/ceph-filesystem-crd.md

```diff
@@ -94,6 +96,8 @@ spec:
     activeStandby: true
 ```
 
+> **NOTE**: For erasure coded, we have to create replicated pool as primary pool and erasureCoded as secondry pool and we need to add a directory layout for a secondary ec data pool [cephfs file-layouts](https://docs.ceph.com/en/latest/cephfs/file-layouts/#adding-a-data-pool-to-the-file-system).
```

Comment:
Since Rook configures the pools and the setfattr is a separate command, the link to the Ceph docs could be confusing. So we probably need to include the complete instructions here in Rook, and just link to Ceph as an fyi.
```suggestion
> **IMPORTANT**: For erasure coded pools, we have to create a replicated pool as the primary pool and erasureCoded as the secondary pool. 

Separately, a directory layout property needs to be added to use the secondary EC data pool. 

<Insert example setfattr command>

See [CephFS file layouts](https://docs.ceph.com/en/latest/cephfs/file-layouts/#adding-a-data-pool-to-the-file-system) for more details.
```

Reply:
right, updated.